providers/summary/scancode.js

```diff
@@ -30,9 +30,12 @@ class ScanCodeSummarizer {
     const buckets = this.computeFileBuckets(harvested.content.files, facets)
     set(result, 'licensed.facets', {})
     const facetsObject = get(result, 'licensed.facets')
+    const declaredLicenses = new Set()
     for (const key in buckets) {
       facetsObject[key] = this.summarizeLicenseInfo(buckets[key])
+      this.summarizeDeclaredLisense(buckets[key], declaredLicenses)
```

Comment:
I was thinking of looping over the files themselves, rather than the buckets. Several reasons. 

1. the bucketization is not relevant. the summarize just loops over the files anyway
1. files can theoretically be in multiple buckets. would be less efficient but should still be correct.